application/util/string.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+
+import re
+
+phone_number_re = re.compile('(\\+\\d+)? \\((\\d\\d\\d)\\) (\\d\\d\\d)-(\\d\\d\\d\\d)')
```

Comment:
what...does this regex do??? i mean there's prior knowledge of what phone numbers look like, I suppose